### DISCUSSION TITLE
Sum of all row items

### DISCUSSION THREAD

**Discussion by CodingRockz** [supabase/supabase#7991] [Questions]
I have a table with several rows. The column field i want is called 'current_value'.

I want to have a function that returns the sum of all current_values in the table.

I want to display this front end as 'total value'.

**Comment by GaryAustin1** [comment]
A quick google shows the Sum function https://www.w3resource.com/PostgreSQL/postgresql-sum-function.php of Postgres.
Then you put that in an SQL function (use the dashboard function editor if starting out to return the sum.
Then call that with an rpc function from the API.  https://supabase.com/docs/reference/javascript/rpc
